### ISSUE TITLE
Can we use a builder image to publish `asciibinder` image

### ISSUE THREAD

**Issue by gaurav-nelson** [openshift-cs/docs-builder#19]
Maybe we can publish another asciibinder image with small size?

Dockerfile
```dockerfile
FROM ruby:3.1.2-alpine3.16 AS builder
RUN apk update && apk add --virtual build-dependencies build-base
RUN gem install ascii_binder

FROM ruby:3.1.2-alpine3.16
COPY --from=builder /usr/local/bundle /usr/local/bundle
CMD ["/user/local/bundle/bin/asciibinder"]
LABEL url="http://www.asciibinder.org" \
      summary="a documentation system built on Asciidoctor" \
      description="Run the asciibinder container image from the local docs repo, which is mounted into the container. Pass asciibinder commands to run the build. Generated files are owned by root." \
      RUN="docker run -it --rm \
          -v `pwd`:/docs:z \
          IMAGE"
WORKDIR /docs
```

The resulting image is 147 MB compared to existing image 778 MB

```
REPOSITORY                                 TAG       IMAGE ID       CREATED          SIZE
quay.io/openshift-cs/asciibinder-alpine    latest    96f27421fd30   2 minutes ago    147MB
quay.io/openshift-cs/asciibinder           latest    78a32c4f59b9   3 months ago     778MB
```

**Comment by wgordon17** [comment]
Referring to https://github.com/openshift-cs/docs-builder/blob/main/asciibinder/Dockerfile? I don't see any reason why not, open a PR please 🙏

**Comment by wgordon17** [comment]
Closed by #24
